src/upcoming-components/Accordion/Accordion.tsx

```diff
@@ -28,17 +23,8 @@ export interface Props {
  *
  * TODO: update this comment with a description of the component.
  */
-export const Accordion = ({
-  children,
-  className,
-  inverted,
-  ...other
-}: Props) => {
-  const componentClassName = clsx(
-    styles['accordion'],
-    inverted && styles['accordion--inverted'],
```

Comment:
I think this class is still in `Accordion.module.css` as well (not `AccordionPanel.module.css`; I see that one just below) https://github.com/chanzuckerberg/edu-design-system/blob/4e8f950ad570a41bdbfd97769285557e9d6104fa/src/upcoming-components/Accordion/Accordion.module.css#L21

Reply:
ahh, i had this one file in my editor still unsaved :) adding to the PR